tools/integration_tests.py

```diff
@@ -125,9 +125,10 @@ def main():
         rmtree(deno_dir)
     os.environ["DENO_DIR"] = deno_dir
 
+    test_names = unittest.TestLoader().getTestCaseNames(TestIntegrations)
     test_names = [
-        test_name for test_name in unittest.TestLoader().getTestCaseNames(
-            TestIntegrations) if not args.filter or args.filter in test_name
+        test_name for test_name in test_names
+        if not args.filter or args.filter in test_name
```

Comment:
👍 

I may have broken this filter stuff in the last commit (it didn't used to work on master), since the --filter is rejected by unittest.main.
Ideally we'd refactor this so that both test.py and test_main supports filtering.

Reply:
I'll check that